From f59877a84fc6883df1153ef223479b26a3e059ae Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Mon, 13 Jun 2016 14:16:19 +0800
Subject: [PATCH 1094/5242] MLK-12901-2 video: mipi_dsi_samsung: add 10msec
 delay after all the pkt write operation.

commit  0e862c90eafad88191f598ee8c371a460d3c3404 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add 10msec delay after all the pkt write operations to let
the data to take effect on the panel's side.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxc/mipi_dsi_samsung.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/video/fbdev/mxc/mipi_dsi_samsung.c b/drivers/video/fbdev/mxc/mipi_dsi_samsung.c
index 38d1a44..95ad7f4 100644
--- a/drivers/video/fbdev/mxc/mipi_dsi_samsung.c
+++ b/drivers/video/fbdev/mxc/mipi_dsi_samsung.c
@@ -220,6 +220,7 @@ static int mipi_dsi_pkt_write(struct mipi_dsi_info *mipi_dsi,
 			return -ETIMEDOUT;
 		}
 	}
+	mdelay(10);
 
 	return 0;
 }
-- 
1.7.9.5

